sound/smooth_swing_v2.h

```diff
@@ -142,41 +167,37 @@ class SmoothSwingV2 : public SaberBasePassThrough {
             STDOUT.print("  mixab: ");
             STDOUT.print(mixab);
             STDOUT.print("  hum_volume: ");
-            STDOUT.println(hum_volume);
+            STDOUT.print(hum_volume);
+            STDOUT.print("  accent_volume: ");
+            STDOUT.println(accent_volume);
           }
-	  if (on_) {
-	    // We need to stop setting the volume when off, or playback may never stop.
-	    A.set_volume(mixhum * mixab);
-	    B.set_volume(mixhum * (1.0 - mixab));
-	  }
           break;
         }
         A.set_volume(0);
         B.set_volume(0);
         state_ = SwingState::OUT;
-
       case SwingState::OUT:
         if (!A.isOff() || !B.isOff()) {
           if (monitor.ShouldPrint(Monitoring::MonitorSwings)) {
             Serial.println("Waiting for volume = 0");
           }
         }
         PickRandomSwing();
+        accent_player_.Free();
```

Comment:
there are still a lot of whitespace changes in this file

Reply:
Apologies, tried hand-ending to fix, but still showing up as 'changes' in the diff even though I can't see differences when looking at the files.